repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -196,6 +205,10 @@ public class EntityGraphMapper {
 
     private static final Set<String> excludedTypes = new HashSet<>(Arrays.asList(TYPE_GLOSSARY, TYPE_CATEGORY, TYPE_TERM, TYPE_PRODUCT, TYPE_DOMAIN));
 
+    Configuration configuration;
+    KafkaNotification kfknotif;
+    int numPartitions = 5; // Total number of partitions in the Kafka topic
```

Comment:
Let's make this a constant for tag-prop events partitions. It can be reused in the `createTopics` method as well.